### PR TITLE
Add debug mode to migration.

### PR DESCRIPTION
**Pull request by fppt**
This mode makes it fail faster and log more direct info to the user.

### CODE REVIEW

File: grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java [hunk 1]
```diff
@@ -223,15 +222,27 @@ private void sendQueriesWhenBatchLargerThanValue(int value) {
...
                 Thread.sleep(500);
             } catch (InterruptedException e) {
-                LOG.error(e.getMessage());
+                printError(e.getMessage());
+            }
+        }
+
+        for (CompletableFuture completableFuture : futures.values()) {
+            try {
+                //TODO: Figure out why a null sometimes gets put in here
```
Comment (aelred) [inline grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java]: haha what

File: grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java [hunk 2]
```diff
@@ -275,41 +286,36 @@ void sendQueriesToLoader(Collection<Query> queries){
...
+                .thenAcceptAsync(onCompletionOfTask);
 
+        // TODO: use an async client
+        futures.put(newStatus.hashCode(), newStatus);
     }
 
-    private void unblock(CompletableFuture<Json> status){
+    private void unblock(){
         blocker.release();
-        futures.remove(status.hashCode());
```
Comment (aelred) [inline grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java]: We should remove the futures at _some_ point, or we'll have a leak. I guess I'd do it in `waitToFinish` after `get`?

File: grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java (hunk 1 above)
Comment (fppt) [reply grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java]: Ooooo wait I did figure it out! Need to remove that comment and check

File: grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java (hunk 2 above)
Comment (fppt) [reply grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java]: Ah yes good point.

**Pull request merged**